bare-metal/elastic-metal/reference-content/shared-responsibility-model.mdx

```diff
@@ -0,0 +1,118 @@
+
+## Product resiliency
+
+### Availability
+
+Bare Metal Products target the following power supplies and connectivity SLOs:
+
+* For Dedibox servers:
+    * Start: 99.9% availability
+    * Pro: 99.95% availability
+    * Core and Store: 99.99% availability
+* For Elastic Metal servers:
+    * Aluminium: 99.9% availability
+    * Beryllium and Lithium: 99.95% availability
+    * Iridium: 99.99% availability
+* For Apple silicon servers:
+    * Mac mini: 99.95% availability
+
+The detailed SLAs measurements and guarantees can be found at [Service Level Agreement for Bare Metal Services](https://www.scaleway.com/en/bare-metal/sla/).
+
+Once the resource is in your hands, we have no access to the machine and therefore no way to monitor its operation.
+In case of any operational issues, we invite you to consult our [documentation](/bare-metal/) or [create a ticket](https://console.scaleway.com/support/tickets/create) with our support team. We will then do our utmost to ensure that you regain access to your resource as quickly as possible.
+
+### Backups and snapshots
+
+You are fully responsible for managing your machine and its data.
+
+It is your responsibility to ensure data redundancy if necessary, by relying on backups or internal server redundancy through solutions such as RAID.
+
+In the event of a disk or a hardware failure, we cannot guarantee that you will regain access to your machine and data. The failure may require a complete replacement of the machine. Therefore, it is your responsibility to ensure in advance that you have a backup or redundancy of your data on another server. 
+**We cannot be held responsible for the loss of your data.**
+
+Backups and snapshots must be configured by you. Please refer to our documentation for assistance in setting up backups for [Dedibox](/dedibox/dedicated-servers/how-to/use-dedibackup-ftp-backup/), [Elastic Metal](/faq/elastic-metal/#how-do-i-back-up-an-elastic-metal-server), and [Apple silicon](/faq/apple-silicon/#how-can-i-back-up-my-mac-mini).
+Restoring snapshots or backups needs to be triggered by you.
+
+## Configuration and version management
+
+### Installation and configuration
+
+We provide a range of Linux, Windows ans macOS server distributions for automatic installation from the Scaleway console. These distributions come with a default configuration designed for standard use cases, ensuring security, usage efficiency, and reliability. During the setup of your machine, you can modify and customize this initial configuration. However, you are responsible for any impact on your server's availability, security, or performance.
+
+Apple Silicon servers, in particular, are enrolled with a Scaleway operated mobile device management server. Removing the enrolment, or any of the associated configuration profiles, may impact Scaleway's ability to monitor & restart your server on your behalf, for instance, in case of power loss.
```

Comment:
```suggestion
Apple Silicon servers, in particular, are enrolled with a Scaleway-operated mobile device management server. Removing the enrolment, or any of the associated configuration profiles, may impact Scaleway's ability to monitor & restart your server on your behalf, for instance, in case of power loss.
```